addons/l10n_il/data/account_tax_template_data.xml

```diff
@@ -355,7 +356,7 @@
     <record id="il_vat_purchase_exempt" model="account.tax.template">
         <field name="sequence">7</field>
         <field name="description">0%</field>
-        <field name="name">VAT exempt</field>
```

Comment:
No need for adding purchases as the tax will already appear only on vendor bills.